src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -68399,6 +68399,111 @@ static void F(object? x, object y)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "u.y").WithLocation(14, 9));
         }
 
+        [Fact]
+        [WorkItem(46206, "https://github.com/dotnet/roslyn/issues/46206")]
+        public void Tuple_Assignment_25()
+        {
+            var source =
+@"using System;
+class Program
+{
+    static void F(object x, string y)
+    {
+        (object, string?) t = new ValueTuple<object?, string>(item2: """", item1: null) { Item1 = x };
+        t.Item1.ToString();
+        t.Item2.ToString();
+    }
+}";
+            var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
+            comp.VerifyDiagnostics(
+                    // (6,31): warning CS8619: Nullability of reference types in value of type '(object?, string)' doesn't match target type '(object, string?)'.
+                    //         (object, string?) t = new ValueTuple<object?, string>(item2: "", item1: null) { Item1 = x };
+                    Diagnostic(ErrorCode.WRN_NullabilityMismatchInAssignment, @"new ValueTuple<object?, string>(item2: """", item1: null) { Item1 = x }").WithArguments("(object?, string)", "(object, string?)").WithLocation(6, 31));
+            comp.VerifyTypes();
```

Comment:
>comp.VerifyTypes(); [](start = 12, length = 19)

It looks like this is not used. (There are no `/*T:...*/` comments in the source.)